### PR TITLE
Move portfolio items into data, and delete posts

### PR DESCRIPTION
**Pull request by ndorfin**
Hey Laura,

As per #20 , here's the first stab at moving the portfolio items into the `_data` directory.
I've also:

- Deleted the old `_posts`
- Added a configuration file for use with [asdf-vm](https://github.com/asdf-vm/asdf) (A version manager for Ruby)
- Removed the `jekyll-feed` gem
- Added `webrick` to the Gem bundle so that Jekyll works with Ruby 3.x
- Updated the bundle to the latest compatible gems
- Added `_site` to the `.gitignore`
- Moved all 'portfolio' images into a subdirectory

Some changes to take note of…

- Notice how the `_layouts/home.html` didn't change that much? 
- The order of the entries in `_data/portfolio.yml` matters. If you wanted to sort by date, you'd have to change the `{% for %}` in the home layout, or re-order the `portfolio.yml`
- If you'd like, I can make this PR point to a new branch you create, so you can test and merge it before you merge to your main branch.
- Don't forget to `bundle install` when you merge this branch.
- I'm using Ruby 3. If it struggles to re-install, let me know what Ruby version (`ruby -v`) you're using, and I'll set this branch up to mimic that.

Let me know if you have any questions.

What I'd recommend next, is you try using the same technique with your Talks. (I'll create a new issue for that)

Cheers,
-S

### CODE REVIEW

**Comment by starshyyne** [comment]
This looks amazing! I'll take a closer look over the weekend and make sure I can get it running on my local. Thank you!

**Comment by ndorfin** [comment]
~~I'll be away this weekend, and should be back at the grindstone on the 16th.~~
Plans changed, I should be here this weekend. Give me a holler if you need help :)
You should have the rights to modify this PR, setting the target branch, etc.

Good luck, and enjoy :D

File: _includes/head.html [hunk 1]
```diff
@@ -23,7 +23,6 @@
   <meta name="theme-color" content="#ffffff"/>
   {%- seo -%}
   <link rel="stylesheet" href="{{ "/assets/main.css" | relative_url }}">
-  {%- feed_meta -%}
```
Comment (starshyyne) [inline _includes/head.html:26]: Thanks for the tidy up, definitely not using this!

File: _layouts/home.html [hunk 2]
```diff
@@ -15,19 +15,19 @@ <h1 class="page-heading">{{ page.title }}</h1>
...
-              <img src="/images/{{ post.imageUrl }}" title="{{ post.title }}" alt="{{ post.imageAlt }}"/>
-            </a>
-          {%- endif -%}
-          <div>
-            <h3>{{ post.title | escape }}</h3>
-            {{ post.content }}
-          </div>
-      </div>
-    {%- endfor -%}
+      {% for project in site.data.portfolio %}
```
Comment (starshyyne) [inline _layouts/home.html:18]: You're right this is very straightforward thank you :)

**Comment by starshyyne** [comment]
This ran perfectly for me first time, awesome thank you :D I will look at doing the same for the talks!

**Pull request merged**